.github/workflows/backport.yml

```diff
@@ -16,15 +16,21 @@ on:
   pull_request_target:
     types: ["labeled", "closed"]
 
+permissions:
```

Comment:
How is this working right now without these permissions?

Reply:
It was the default without permissions. They are starting to change it. For example updating github workflows isn't possible without this.